src/partitionAccessory.ts

```diff
@@ -137,10 +138,21 @@ export class EnvisalinkPartitionAccessory {
             case 'entrydelay':
             case 'arming':
             case 'busy':
+            case 'troubleledoff':
```

Comment:
Why do we need separate handling for `troubleledoff` and `troubleledon`? These blocks are the same as the new default block below.

Reply:
Yes.  This PR adds explicit support for these two messages.  They should not be allowed to fall through to default handling because if that ever changes then it breaks desired handling of these two messages.  And our explicit handling is not the same as default (new or old).  Specifically we allow `troubleledoff` to go through silently, but `troubleledon` we send an _informational_ message to log.  This is not the same as default which (new version) sends a _warning_ message to log.